scripts/plot_storage_levels.py

```diff
@@ -0,0 +1,144 @@
+    header=[0],
+    index_col=[0],
+)
+colors_csv = colors_csv.T
+colors_odict = OrderedDict()
+for i in colors_csv.columns:
+    colors_odict[i] = colors_csv.loc["Color", i]
+
+
+def plot_on_axes(ax, df, colors_odict=colors_odict):
+    for i in df.columns:
+        ax.plot(
+            df.index,
+            df[i],
+            label=i,
+            linewidth=2,
+            color=colors_odict[i],
+        )
+
+
+def plot_storage_levels(df_dict, colors_odict=colors_odict):
+    r"""
+    Reads in the storage level time series from a dataframe and plots them according to their busses.
+
+    Parameters
+    ----------
+    df_dict : dictionary of pandas.DataFrame()
+        List with 1-3 dataframes, each for one bus,
+         with storage level time series for a single country.
+
+    Returns
+    -------
+    figure
+    """
+    if len(df_dict.items()) > 1:
+        fig, (ax1, ax2) = plt.subplots(nrows=2, ncols=1, figsize=(14, 5), linewidth=20)
+    else:
+        fig, (ax1) = plt.subplots(nrows=1, ncols=1, figsize=(14, 5), linewidth=20)
+
+    plot_on_axes(ax1, df_dict["df_elec"])
+    ax1.legend(loc="upper left")
+    ax1.set_ylabel("Electricity [GWh]")
+    ax1.tick_params(axis="y", labelcolor=colors_odict["BAT"])
+
+    if "df_heat" in df_dict.keys():
+        plot_on_axes(ax2, df_dict["df_heat"])
+        ax2.legend(loc="upper left")
+        ax2.set_ylabel("Heat [GWh]")
+
+    if "df_h2" in df_dict.keys():
+        ax3 = ax1.twinx()
+        ax3.set(ylim=(0, 45))
+        #        df_dict["df_h2"] = df_dict["df_h2"] / 1000 # conversion from GWh to TWh
+        plot_on_axes(ax3, df_dict["df_h2"])
+        ax3.set_ylabel("Electricity [TWh]")
+        ax3.legend(loc="upper right")
+        ax3.tick_params(axis="y", labelcolor=colors_odict["H2 cavern"])
+
+    return fig
+
+
+if __name__ == "__main__":
+    paths = Dict()
+    paths.postprocessed = sys.argv[1]
+    paths.plotted = sys.argv[2]
+    if not os.path.exists(paths.plotted):
+        os.makedirs(paths.plotted)
+    timeseries_directory = os.path.join(
+        paths.postprocessed, "oemoflex-timeseries/variable"
+    )
+    # oemof.solph 0.3.2 used 'capacity' to name this variable, oemof.solph > 0.4.0 renamed it to 'storage level'.
+    storage_level_data = os.path.join(timeseries_directory, "capacity.csv")
+    capacities = pd.read_csv(
+        storage_level_data, header=[0, 1, 2], parse_dates=[0], index_col=[0]
+    )
+
+    # TODO: import this from user_definitions.py on top of this script
+    timeframe = [
+        ("2019-01-01 00:00:00", "2019-01-31 23:00:00"),
+        ("2019-07-01 00:00:00", "2019-07-31 23:00:00"),
+    ]
+    regions = ["DE", "PL"]
+
+    for timeframe, region in itertools.product(timeframe, regions):
+        df = pd.DataFrame()
+        for column in capacities.columns:
+            if region in column[0]:
+                df[column] = capacities.loc[:, column]
+        df = df / 1000  # conversion from MWh to GWh
```

Comment:
Please define a constant `CONV_NUMBER`

Reply:
Done in b86c4fe . But I am not sure whether I should define a second CONV_NUMBER for hydrogen, which I want to plot in TWh. For now, I have left the division by 1000 there explicit.